.github/workflows/enos-run.yml

```diff
@@ -137,15 +138,33 @@ jobs:
           path: ./enos/support/downloads
       - name: Unzip and rename Boundary bundle
         run: |
-          unzip ${{steps.download.outputs.download-path}}/*.zip -d enos/support
+          unzip ${{steps.download.outputs.download-path}}/*.zip -d enos/support/boundary
           mv ${{steps.download.outputs.download-path}}/*.zip enos/support/boundary.zip
+      - name: Set up node for e2e UI tests
+        uses: actions/setup-node@v3
+        if: matrix.filter == 'e2e_ui builder:crt'
+        with:
+          node-version: '16.x'
+      - name: Checkout boundary-ui
+        uses: actions/checkout@v3
+        if: matrix.filter == 'e2e_ui builder:crt'
+        with:
+          repository: hashicorp/boundary-ui
+          ref: moduli-e2e-tests-ICU-7418 # !! CHANGE TO main
```

Comment:
Tests currently live in this POC branch. Update this line once the `boundary-ui` PR is merged.